dashboard/settings.py

```diff
@@ -35,6 +35,9 @@
 # Application definition
 
 INSTALLED_APPS = [
+    'dashboard.socialaccount.providers.fxa',
+    'landing',
+
```

Comment:
`{% extends "base.html" %}` was extending the allauth *base.html* otherwise. If there's another way to favor our *base.html* I can do that instead.